<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="palette">
    <palette>
     <active/>
     <inactive/>
     <disabled/>
    </palette>
   </property>
   <widget class="QPushButton" name="selectImpushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="picturelabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>351</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>260</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tag</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>300</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Comment</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tagLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>260</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="commentLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>300</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>350</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Search:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuLanguage_2">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="separator"/>
     <addaction name="separator"/>
     <addaction name="actionHungarian_2"/>
     <addaction name="actionEnglish_2"/>
     <addaction name="actionSpanish"/>
    </widget>
    <widget class="QMenu" name="menuStyle_2">
     <property name="title">
      <string>Style</string>
     </property>
     <addaction name="actionDefault_2"/>
     <addaction name="actionDark_2"/>
     <addaction name="actionSunny_2"/>
     <addaction name="actionBlue_2"/>
    </widget>
    <addaction name="menuLanguage_2"/>
    <addaction name="menuStyle_2"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionBlue">
   <property name="text">
    <string>Blue</string>
   </property>
  </action>
  <action name="actionSunny">
   <property name="text">
    <string>Sunny</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionHungarian_2">
   <property name="text">
    <string>Hungarian</string>
   </property>
  </action>
  <action name="actionEnglish_2">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionSpanish">
   <property name="text">
    <string>Spanish</string>
   </property>
  </action>
  <action name="actionDefault_2">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionDark_2">
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionSunny_2">
   <property name="text">
    <string>Sunny</string>
   </property>
  </action>
  <action name="actionBlue_2">
   <property name="text">
    <string>Blue</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
